### PR TITLE
pkgbuild revision

### PR DESCRIPTION
**Pull request by FabioLolix**
linux-enable-ir-emitter don't have stable release so need to be re-uploaded as linux-enable-ir-emitter-git

### CODE REVIEW

**Comment by EmixamPP** [comment]
The problem is that the maintainer of this package never announced himself :/

File: AUR/PKGBUILD [hunk 1]
```diff
@@ -2,39 +2,38 @@
 # Maintainer: Antoine Bertin <ant.bertin@gmail.com>
 # Maintainer: Andrey Kolchenko <andrey@kolchenko.me>
 
-pkgname=linux-enable-ir-emitter
-pkgver=20210725.0
+pkgname=linux-enable-ir-emitter-git
+pkgver=r167.04f6c73
 pkgrel=1
 pkgdesc='Enables infrared cameras that are not directly enabled out-of-the box.'
+arch=(x86_64)
```
Comment (komex) [inline AUR/PKGBUILD:9]: Can you please quote all values?

File: AUR/PKGBUILD [hunk 2]
```diff
@@ -2,39 +2,38 @@
...
 url='https://github.com/EmixamPP/linux-enable-ir-emitter'
 license=(MIT)
-arch=(x86_64)
-
+depends=(python python-opencv python-yaml)
+makedepends=(git)
```
Comment (komex) [inline AUR/PKGBUILD:13]: Why did you remove `gcc` and `make`?

**Review by komex — COMMENTED** [review]
@EmixamPP, are you going to use releases in the future? May be you release a stable version and in AUR I will maintain two packages (stable and git)?

**Comment by EmixamPP** [comment]
I would say that, from now on, we can operate in release. The program seems to be stable, so I don't intend to do any more "express updates".

And furthermore, I have no more idea what features to add or improve.

**Comment by komex** [comment]
> I would say that, from now on, we can operate in release. The program seems to be stable, so I don't intend to do any more "express updates".
> 
> And furthermore, I have no more idea what features to add or improve.

So, git version in AUR is not needed.

File: AUR/PKGBUILD (hunk 2 above)
Comment (FabioLolix) [reply AUR/PKGBUILD:13]: gcc and make are part of base-devel and don't need to added to makedepends

File: AUR/PKGBUILD (hunk 1 above)
Comment (FabioLolix) [reply AUR/PKGBUILD:9]: Quoting in arch=() license=() depends=(), etc is effectively pointless when not needed

Comment (FabioLolix) [reply AUR/PKGBUILD:9]: If for you quoting everything make the pkgbuild more readable, to me have the opposite effect

**Comment by FabioLolix** [comment]
linux-enable-ir-emitter-git has been update (also I didn't see it existed when opening the PR here) https://aur.archlinux.org/cgit/aur.git/commit/?h=linux-enable-ir-emitter-git&id=826d298f8307d245dfaf722d8fdf5a6c4aa96633

added EmixamPP  as maintainer and orphaned it

**Comment by EmixamPP** [comment]
Thank you so much for your help!
